app/validators/national_insurance_number_format_validator.rb

```diff
@@ -2,8 +2,50 @@ class NationalInsuranceNumberFormatValidator < ActiveModel::EachValidator
   NINO_REGEX_FILTER = /\A[A-Z]{2}[0-9]{6}[A-D]{1}\Z/
 
   def validate_each(record, attribute, value)
-    return if value.gsub(/\s/, "").upcase.match?(NINO_REGEX_FILTER)
+    normalised_value = value.gsub(/\s/, "").upcase
```

Comment:
Should we do this in the validation? Seems like something the form/model should do before calling the validation

Reply:
after having a think and looking around, i think the line isn't needed and I was just assuming it would come in badly
```
NINO_REGEX_FILTER = /\A[A-Z]{2}[0-9]{6}[A-D]{1}\Z/
```
The regex should cater for whitespace and casing so I'm thinking I'm just gonna remove highlighted line and use `value`